packages/fluxible-addons-react/connectToStores.js

```diff
@@ -59,6 +59,7 @@ function createComponent(Component, stores, getStateFromStores, customContextTyp
     });
 
     hoistNonReactStatics(StoreConnector, Component);
+    StoreConnector.WrappedComponent = Component;
```

Comment:
When do you expect this to be merged/published? Did you want me to the attaching of `WrappedComponent` in the other files?